src/fusor/models.py

```diff
@@ -216,24 +212,11 @@ class LinkerElement(BaseStructuralElement, extra="forbid"):
     type: Literal[FUSORTypes.LINKER_SEQUENCE_ELEMENT] = (
         FUSORTypes.LINKER_SEQUENCE_ELEMENT
     )
-    linker_sequence: SequenceDescriptor
-
-    @field_validator("linker_sequence", mode="before")
```

Comment:
we don't need any validation in this class anymore, since pydantic will throw a value error if the passed in `linkerSequence` is not a `LiteralSequenceExpression`